formats/bibtex/index.js

```diff
@@ -0,0 +1,22 @@
+import Format from "../../src/format.js";
+import { BibLatexParser, BibLatexExporter } from "https://esm.sh/biblatex-csl-converter/lib/index.js";
+
+export default class BibTeX extends Format {
+	static defaultOptions = {
+		processUnexpected: true,
+		processUnknown: true,
+	};
+
+	static extensions = ["bib"];
+	static mimeTypes = ["text/plain"];
+
+	static parse (str, options) {
+		options = this.resolveOptions(options, "parse");
+		return new BibLatexParser(str, options).parse();
```

Comment:
IIRC this does not produce a useful format by itself, though it preserves duplicate keys which could be useful for some use cases like reference managers. 
We should probably have some sort of option to get back an object of ids to metadata. And something to transform to CSL, though I'm not sure what that entails exactly. But since the functionality is there, we could provide an option.